dpnp/tests/test_manipulation.py

```diff
@@ -87,6 +89,12 @@ def test_size():
     assert dpnp.size(a, 0) == exp
     assert dpnp.size(ia, 0) == exp
 
+    assert dpnp.size(ia, 1) == numpy.size(a, 1)
```

Comment:
I think it makes sense to create a separate `test_size_tuple()` or add a condition to check numpy version inside to keep testing for `int` if numpy< 2.4 in the future